src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -598,6 +598,7 @@ public override void OnEditorSpeciesSetup(Species species)
         // that just marks things dirty and we update our stats on the next _Process call)
         Membrane = Editor.EditedCellProperties!.MembraneType;
         Rigidity = Editor.EditedCellProperties.MembraneRigidity;
+        originalRigidity = (int)Math.Round(Rigidity * Constants.MEMBRANE_RIGIDITY_SLIDER_TO_VALUE_RATIO);
```

Comment:
Storing the original value like this goes against the dynamic MP, so having just the rigidity not follow the way everything else is handled in the editor is not the right design in my opinion.